docs-chef-io/content/inspec/troubleshooting.md

```diff
@@ -92,3 +92,23 @@ Chef licensing data is stored on the `$HOME/.chef/licenses.yaml` file.
 The `licenses.yaml` file is malformed and corrupt.
 
 Restore the file content to its original state to resolve this issue.
+
+## Unable to resolve dependency: user requested 'inspec-{RESOURCE_NAME}-resources (> 0)'
+
+Starting from **InSpec version 7**, Chef InSpec profiles can include dependencies on gem-based resource packs. If this error occurs, it can be resolved through the following steps:
```

Comment:
Replace it with: Chef InSpec profiles, which can include dependencies on gem-based resource packs from **InSpec version 7**. If this error occurs, resolve it using the following steps:

Reply:
Chef InSpec profiles can include dependencies on gem-based resource packs from InSpec version 7. If this error occurs, resolve it using the following steps: